---
xbee_address:
  base_station: 417E628A
  husky1: 417E6E0E
  husky2: 41AF1C55  
  jackal1: 417E6274
  jackal2: 41AF1A01
